unittests/main.cpp

```diff
@@ -33,8 +33,8 @@ boost::unit_test::test_suite* init_unit_test_suite(int argc, char* argv[]) {
    if(is_verbose) {
       fc::logger::get(DEFAULT_LOGGER).set_log_level(fc::log_level::debug);
    } else {
-      fc::logger::get(DEFAULT_LOGGER).set_log_level(fc::log_level::off);
-   }
+      fc::logger::get(DEFAULT_LOGGER).set_log_level(fc::log_level::debug);
```

Comment:
@AndriiAndrusyshyn I'm not 100% sure, but in latest develop such changes already done